conan/cli/commands/create.py

```diff
@@ -56,6 +60,13 @@ def create(conan_api, parser, *args):
     lockfile = conan_api.lockfile.update_lockfile_export(lockfile, conanfile, ref, is_build)
 
     print_profiles(profile_host, profile_build)
+    if profile_host.runner and not os.environ.get("CONAN_RUNNER_ENVIRONMENT"):
+        return {
+            'docker': DockerRunner,
+            'ssh': SSHRunner,
+            'wsl': WSLRunner,
+        }[profile_host.runner.get('type')](conan_api, 'create', profile_host, profile_build, args, raw_args).run()
```

Comment:
```suggestion
        try:
            runner_class = { 'docker': DockerRunner, 'ssh': SSHRunner, 'wsl': WSLRunner}[profile_host.runner.get('type')]
        except KeyError:
            raise ConanException(f"Unrecognized runner 'type'={profile_host.runner.get('type')}")
        runner = runner_class(conan_api, 'create', profile_host, profile_build, args, raw_args)
        return runner.run()
```
A bit more UX-friendly ready, as users will miss or mistype the ``type``.